### DISCUSSION TITLE
在切出游戏回到桌面的时候，magpie闪退了

### DISCUSSION THREAD

**Discussion by ilikepython3** [Blinue/Magpie#1015] [Q&A] [answered]
请问如果解决这个问题？

**Comment by Blinue** [comment]
请提供日志

**Reply by ilikepython3** [reply]
[magpie.log](https://github.com/user-attachments/files/17543999/magpie.log)
我在刚刚又运行了一次并且闪退，但是没有看到日志最新的记录，需要附上视频吗？

**Reply by ilikepython3** [reply]
https://github.com/user-attachments/assets/9eeebe93-e8e8-43de-9533-bfbe4293bfe2

这是运行崩溃的视频文件

**Reply by Blinue** [reply]
你有用什么第三方软件吗，右下角有帧率显示。还有这个崩溃只在这一个游戏里出现吗？

**Reply by ilikepython3** [reply]
帧率显示是steam的自带功能，以及这个崩溃不止在这一个游戏中出现，我又试了几个也有这种情况。

**Reply by ilikepython3** [reply]
帧率显示关了这个问题依旧存在，以及将UI界面关闭，仅留托盘图标，也还是会在切回桌面几秒后自动退出程序。

**Reply by Blinue** [reply]
在系统中搜索“事件查看器”，进入"Windows日志"->“应用程序”，里面会有崩溃记录
![image](https://github.com/user-attachments/assets/f5b3c695-f9c5-4135-bd6a-685bdf5937d9)

**Reply by ilikepython3** [reply]
<img width="1280" alt="magpie_error" src="https://github.com/user-attachments/assets/7d63eada-a5fc-4244-a743-f4776aebe5c2">
很抱歉回复这么慢，这是结果

**Reply by ilikepython3** [reply]
看起来我是需要更新驱动或者更新Windows之类的？

**Reply by Blinue** [reply, marked as answer]
先试试更新驱动。nvppex.dll 似乎是用于N卡滤镜 citizenfx/fivem#2441，你把滤镜关掉试试。

**Reply by ilikepython3** [reply]
很感谢你的答复，问题得到解决了，我把滤镜选项关掉就可以了，不用更新驱动
